src/main.rs

```diff
@@ -104,6 +108,7 @@ Options:
     -i, --iface <iface>      Specify the local IP interface.
     -t, --tunnel <tunnel>    Set the tunnel endpoint's hostname. If omitted, the tunnel is disabled.
     -s, --tunnel-secret <secret>    Set the tunnel shared secret. [default: secret]
+        --disable-tls               Run as a plain HTTP server, disabling encryption.
     -u, --remote-name <hostname>    Set remote hostname. This the URL to access the box through the bridge. If omitted, the tunnel is disabled
```

Comment:
I didn't know you could have options without the simplified version :) Could you take the chance to also remove the `-u` option, please? Thanks!